CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## 4.1.1
+## 4.1.2
+- Remove ES 1.x from test matrix. The testing requirements were too severe, and update bugs were too troublesome to correct. See more about the update bugs @ https://github.com/elastic/elasticsearch/issues/19103
```

Comment:
@jordansissel Good point. I'll bump it to 5.0.0. 

I'm not sure exactly some update settings stopped working. Is it worth it to try to untangle that ball? The git commit goes back to April when I checked in the [commit](https://github.com/logstash-plugins/logstash-output-elasticsearch/commit/77c437b9ab8ddc0f3397943a9d36426bc1002ead) that made tests always green.